games/minetest/Portfile

```diff
@@ -85,20 +103,22 @@ patchfiles-append       003-patch-ignore-psn-option-mac-bundle.diff
 
 configure.args-append   -DCMAKE_BUILD_TYPE=Release \
                         -DCMAKE_INSTALL_PREFIX:PATH=${applications_dir} \
-                        -DBUILD_CLIENT=1 \
-                        -DENABLE_SOUND=1 \
-                        -DENABLE_REDIS=1 \
-                        -DENABLE_POSTGRESQL:BOOL=OFF \
-                        -DENABLE_LEVELDB=1 \
-                        -DENABLE_FREETYPE=1 \
-                        -DENABLE_CURL=1 \
-                        -DENABLE_GETTEXT=1 \
-                        -DENABLE_SYSTEM_JSONCPP=1 \
-                        -DENABLE_UPDATE_CHECKER=0 \
+                        -DBUILD_UNITTESTS=OFF \
+                        -DBUILD_CLIENT=ON \
+                        -DBUILD_SERVER=ON \
+                        -DENABLE_SOUND=ON \
+                        -DENABLE_REDIS=OFF \
+                        -DENABLE_POSTGRESQL=OFF \
+                        -DENABLE_LEVELDB=ON \
+                        -DENABLE_FREETYPE=ON \
+                        -DENABLE_CURL=ON \
+                        -DENABLE_GETTEXT=ON \
+                        -DENABLE_SPATIAL=ON \
+                        -DENABLE_SYSTEM_GMP=ON \
+                        -DENABLE_SYSTEM_JSONCPP=ON \
                         -DCURSES_INCLUDE_PATH:FILEPATH=${prefix}/include \
                         -DENABLE_LUAJIT=ON \
-                        -DLUA_INCLUDE_DIR:PATH=${prefix}/include/luajit-2.1 \
-                        -DLUA_LIBRARY:FILEPATH=${prefix}/lib/libluajit-5.1.dylib
```

Comment:
why is this not needed anymore - does it find the needed MacPorts file automatically?

Reply:
iirc when i wrote this up a few yrs ago, it would find the wrong files if lua was installed

Reply:
YES - indeed

This like at least another config line in the legacy Portfile was just making life complicated.

Over time the CMake was improved by the MT devs in a way that the new addition of an explicit `-DENABLE_LUAJIT=ON` finds the needed MacPorts file automatically. Thus the addition of `port:luajit` to the new Portfile.

Please note this was tested by my on my Mojave platform with a clean install already in the past.

One may refer to #17080 and section on Portfile improvements below the

> Particular POIs are marked with `<-- # better compatibility` or similar comment inline.

for an overall perspective.

Reply:
@kencu - THX

> In [games/minetest/Portfile](https://github.com/macports/macports-ports/pull/17077#discussion_r1058733221):

```
                          -DCURSES_INCLUDE_PATH:FILEPATH=${prefix}/include \
                         -DENABLE_LUAJIT=ON \
-                        -DLUA_INCLUDE_DIR:PATH=${prefix}/include/luajit-2.1 \
-                        -DLUA_LIBRARY:FILEPATH=${prefix}/lib/libluajit-5.1.dylib
```
>  iirc when i wrote this up a few yrs ago, it would find the wrong files if lua was installed

I would like to take the opportunity to express my sincere thanks to @kencu in kindly maintaining the `minetest` port in the past apparently. His reply reached my by email and nicel adds to my assumptions above.

Resolved as before.

Reply:
@Zweihorn you'll find that @kencu is typically right about these type of things 😉. 

Can you please verify that the resulting minetest binaries link with libraries provided by MacPort port(s) and not possible macOS versions?

Reply:
OK - re-introducing the two "-DLUA_ ..." options.

I trust @kencu and you. As the options provide a safety net for the compile they cannot be redundant .